doc/manual/calling-c-and-fortran-code.rst

```diff
@@ -584,7 +578,7 @@ Mapping C Functions to Julia
 ccall/cfunction argument translation guide
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-For translating a ``c`` argument list to ``Julia``:
```

Comment:
Since you're fixing details: I don't think the quotes are needed here, they are even confusing.